<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>581</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Обрати файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>291</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Скасувати</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filePathLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Шлях до файлу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>291</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Додати</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
